static/_redirects

```diff
@@ -57,3 +57,4 @@
 /docs/internationalization_index /docs/internationalization
 /docs/internationalization_translation /docs/translations
 /docs/lovelace_custom_card /docs/frontend/custom-ui/lovelace-custom-card
+/docs/lovelace_register_resource /docs/frontend/custom-ui/registering_resource
```

Comment:
```suggestion
```

We only need the `_redirects` when changing the structure of the site to make sure links from external sites don't go dead. As this is a new file there is no need for a redirect.